### DISCUSSION TITLE
Regarding styling

### DISCUSSION THREAD

**Discussion by nilavrusty** [csandman/chakra-react-select#103] [Q&A] [answered]
**My question is...**
I want to change colors & other css properties, even classNamePrefix of react-select isn't working with the component, menu list doesn't get the prefix

**Comment by csandman** [comment]
This has been asked in #2 and I'm still working on a good solution for passing custom styles through to the inner components in a consistent way.  As for `classNamePrefix`, there is no way to prefix the generated chakra classNames without altering the webpack config, so I'd need to figure out an alternative there.  One thing I could do is add a custom className to each of the subcomponents, similar to how the original package does, and apply the `classNamePrefix` to that.  I'll definitely look into it!

**Comment by csandman** [comment, marked as answer]
So I fixed the `classNamePrefix` issue in [1.4.0](https://github.com/csandman/chakra-react-select/releases/tag/v1.4.0), and considering that is the main difference between this issue and #2 I'm going to mark this as closed for now.  In the process, I ended up adding all of the default [classNames that react-select uses](https://react-select.com/styles#using-classnames) so you can style components using those.  I am however, still working on a more Chakra-ey method of styling components over using raw css to style components using classNames, but I will comment on #2 when there is progress on that. Hopefully this can offer a good enough solution in the meantime however. 

Let me know if you're still having any issues with it!

I've also updated the example to use the newest version so you can see them here: https://codesandbox.io/s/chakra-react-select-demo-65ohb?file=/package.json

Side note, if you want to inspect the specific components in the select menu to see their classNames without the menu closing on you, the easiest way is to use the [Emulate a focused page](https://stackoverflow.com/a/64456947) option in Chrome's devtools. Enable it, then open the select menu and use the element selector (top left of devtools) to select an item in the menu.  Then you'll be free to explore the DOM structure of the opened select menu without it closing automatically.  If you're doing this with the CodeSandbox demo however, it only works if you open the sandbox in its own page: https://65ohb.csb.app/
